diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter1/EnumValueCaseTest.kt

```diff
@@ -1,2 +1,16 @@
 package org.cqfn.diktat.ruleset.chapter1
 
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.rules.IdentifierNaming
+import org.cqfn.diktat.util.FixTestBase
+import org.junit.Test
+
+class EnumValueCaseTest : FixTestBase ("test/paragraph1/naming", IdentifierNaming(),
+        listOf(
+                RulesConfig("ENUM_VALUE", true, mapOf()))
```

Comment:
You don't need to specify rules config list, if you don't need to test specific flags. Also, you should have used `Warnings.ENUM_VALUE.name` instead of hard-coded string.